.github/label-pr-config.yml

```diff
@@ -0,0 +1,83 @@
+autolabeler:
+  - label: 'Type: documentation'
+    files:
+      - '*.md'
+    # Checks there is no word character (alphanumeric & underscore) before target words, or after to avoid accidental matches
+    title:
+      - '/(?<![\w])((doc)|(docs)|(documentation))(?![\w])/i'
+  - label: 'Type: fix'
+    branch:
+      - '/fix\/.+/'
+    title:
+      - '/(?<![\w])((fix)|(fixes)|(fixed)|(bug)|(bugfix)|(bug fix))(?![\w])/i'
```

Comment:
nit: `bug fix` is unnecessary as you are already matching on `bug` and `fix`.

Reply:
Updated to remove redundant term!